### PR TITLE
Add AppConfig values for VPN connections

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: source/mobile/mobile-appconfig.rst [hunk 1]
```diff
@@ -36,6 +36,10 @@ The following table shows all the configuration options that can be sent from th
...
+| useVPN                 | Will the app connect to the Mattermost Server using a per-app VPN or VPN on-demand.                                                                                                                             | String | ``false``     | ``true | false`` | Yes         | No                       |
++------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
+| timeoutVPN             | For how long the request will wait (in seconds) for the VPN connection to establish before timeout.                                                                                                             | String | 30            |                  | Yes         | No                       |
```
Comment (jasonblais) [inline source/mobile/mobile-appconfig.rst]: Proposed description:

```
Set how long the request waits (in seconds) for a VPN connection to establish before timeout.
```

**Comment by enahum** [comment]
@jasonblais changes made

File: source/mobile/mobile-appconfig.rst [hunk 2]
```diff
@@ -36,6 +36,10 @@ The following table shows all the configuration options that can be sent from th
...
 | username               | Set the username or email address to use to authenticate against the Mattermost Server.                                                                                                                         | String |               |                  | Yes         | Yes                      |
 +------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
+| useVPN                 | Enable connection to the Mattermost Server to ue a per-app VPN or VPN on-demand.                                                                                                                                | String | ``false``     | ``true | false`` | Yes         | No                       |
```
Comment (hmhealey) [inline source/mobile/mobile-appconfig.rst]:
```suggestion
| useVPN                 | Enable connection to the Mattermost Server to use a per-app VPN or VPN on-demand.                                                                                                                                | String | ``false``     | ``true | false`` | Yes         | No                       |
```

File: source/mobile/mobile-appconfig.rst [hunk 3]
```diff
@@ -36,6 +36,10 @@ The following table shows all the configuration options that can be sent from th
...
 +------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
+| useVPN                 | Enable connection to the Mattermost Server to use a per-app VPN or VPN on-demand.                                                                                                                               | String | ``false``     | ``true | false`` | Yes         | No                       |
++------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+---------------+------------------+-------------+--------------------------+
```
Comment (esethna) [inline source/mobile/mobile-appconfig.rst]: @enahum do we need to add this to any other places in the docs? ie instructions on using a VPN connection?

Comment (enahum) [reply source/mobile/mobile-appconfig.rst]: This is the only place we have documentation about this. What are you thinking?

Comment (esethna) [reply source/mobile/mobile-appconfig.rst]: Wondering if we should have a section in this doc for setting up VPN? But let's merge this for now and then we can discuss

**Comment by esethna** [comment]
@amyblais ready to merge once 1.17 is released

**Comment by esethna** [comment]
Merging now so we can share docs with customers

**Pull request merged**